pkg/workflows/workflows.go

```diff
@@ -47,17 +47,17 @@ func CreateWorkflows(dest string, config *WorkflowConfig) error {
 	if err = updateProductionDeployments(deployType, dest, config); err != nil {
 		return err
 	}
-	workflow, ok := deployNameToWorkflow[deployType]
+	workflowType, ok := deployNameToWorkflow[deployType]
 	if !ok {
 		return errors.New("unsupported deployment type")
 	}
 
-	workflowTemplate := getWorkflowFile(workflow)
+	workflowTemplate := getWorkflowFile(workflowType, workflows)
 
 	replaceWorkflowVars(deployType, config, workflowTemplate)
 
-	ghWorkflowPath := dest + "/.github/workflows/"
-	ghWorkflowFileName := ghWorkflowPath + workflowFilePrefix + workflow.workflowFileSuffix + ".yml"
+	ghWorkflowPath := path.Join(dest, ".github", "workflows")
+	ghWorkflowFileName := path.Join(ghWorkflowPath, workflowFilePrefix+workflowType.workflowFileSuffix+ymlExtension)
```

Comment:
💡☕️ Suggestion please: What is `workflowFilePrefix+workflowType` context? or `workflowFileSuffix+ymlExtension` if the end result will be something like `someyaml.yaml` we could simplify by extracting it renaming it to logical and constructing it in more simple way like:

```*.go
resultdomelogicalname := fmt.Sprintf("%s%s%s%s", str1, str2, str3, str4) 
```